sql/mysqld.h

```diff
@@ -174,15 +174,34 @@ extern char *opt_backup_history_logname, *opt_backup_progress_logname,
             *opt_backup_settings_name;
 extern const char *log_output_str;
 extern const char *log_backup_output_str;
-extern char *temporal_current_timestamp;
-extern my_bool vers_force;
+
+enum vers_range_type_t
+{
+  FOR_SYSTEM_TIME_UNSPECIFIED = 0,
+  FOR_SYSTEM_TIME_AS_OF,
+  FOR_SYSTEM_TIME_FROM_TO,
+  FOR_SYSTEM_TIME_BETWEEN,
+  FOR_SYSTEM_TIME_ALL,
+  FOR_SYSTEM_TIME_BEFORE
+};
+
+struct st_vers_current_time
+{
+  char *str_value; // must be first
+  vers_range_type_t type;
+  MYSQL_TIME ltime;
+  st_vers_current_time() :
```

Comment:
The structure complies with c++0x definition of POD (see [this](http://stackoverflow.com/a/1130194/3824341) answer). I would prefer to [disable](http://stackoverflow.com/questions/3378560/how-to-disable-gcc-warnings-for-a-few-lines-of-code) this particular warning.